.github/workflows/gcp-deploy.yaml

```
@@ -0,0 +1,46 @@
+              sudo docker-compose down
+            else
+              echo "docker-compose.yml not found in ${{ env.REMOTE_APP_PATH }}"
+            fi
+
+            echo "Pulling the latest changes from Git..."
+            git pull origin ${{ github.event.inputs.branch || github.ref_name }}
+
+            echo "Rebuilding Docker Compose services..."
+            sudo docker-compose up -d --build
+            echo "Deployment successful!"
+          '
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)